scene/gui/line_edit.h

```diff
@@ -259,26 +259,26 @@ class LineEdit : public Control {
 	void set_placeholder_alpha(float p_alpha);
 	float get_placeholder_alpha() const;
 
-	void set_cursor_position(int p_pos);
-	int get_cursor_position() const;
+	void set_caret_position(int p_pos);
+	int get_caret_position() const;
 
 	void set_max_length(int p_max_length);
 	int get_max_length() const;
 
-	void append_at_cursor(String p_text);
+	void append_at_caret(String p_text);
```

Comment:
Would suggest `insert_text_at_caret` to align with `TextEdit`.

Reply:
Done.